### ISSUE TITLE
Right Click Only Option for OffhandGapple

### ISSUE THREAD

**Issue by l1ving** [kami-blue/client#1748]
Suggested by <@!322183265463304192>.

Option so that your totem is always out but when holding right click with a sword, a gapple will go to your offhand and be eaten. This worked in a previous version (v1.1.6 release) but currently does not. Video showing how it worked in that version https://youtu.be/iZvNnxmPVyA

**Comment by Xiaro** [comment]
AutoOffhand is designed to be only switching items or switching to totem when **needed** to have ability to manually override item in offhand. Check weapon under gapple should does the similar thing already.

**Comment by l1ving** [comment]
I only left the description of the original suggestion as a courtesy, I've seen multiple people want the old behavior of right click to gap for whatever reason.

**Comment by Xiaro** [comment]
> I only left the description of the original suggestion as a courtesy, I've seen multiple people want the old behavior of right click to gap for whatever reason.

🗿 It is just gonna break compatibility and anticheat doesn't like switching items fast as well

**Comment by l1ving** [comment]
Plugins are able to annotate if they're incompatible with a module and then not allow being enabled at the same time.

**Comment by Xiaro** [comment]
If it is incompatible with AutoOffhand then there is no point of having it because AutoOffhand is an essential module for pvp

**Comment by iBuyMountainDew** [comment]
@l1ving I never explicitly said **that** to be the case. Plugins will have to handle that themselves. I implore you to stop thinking of plugins as wrappers for _single modules_.

**Comment by l1ving** [comment]
I clearly meant modules

**Comment by iBuyMountainDew** [comment]
Ah fair, but to be quite honest it was not clear. Don't try and tell me I'm the cheeser in English when you're interpreting "said context" as literally "this person has previously said the word context".

**Comment by Xiaro** [comment]
The reason of having totem always out is just preventing death. AutoOffhand already does it by checking the health and potential damage from enemy. Having OffhandGapple and CheckWeapon on will put a gapple in your hand when you holding a sword. And switching back to totem when you are in risk of death.

**Comment by l1ving** [comment]
Some people still prefer manually controlling it. I'm not here to explain it, it's a suggestion for a plugin and it doesn't hurt anybody to add it as a plugin.

**Comment by iBuyMountainDew** [comment]
Me and @l1ving did agree on this. The issue will remain open.

**Comment by Xiaro** [comment]
> Some people still prefer manually controlling it. I'm not here to explain it, it's a suggestion for a plugin and it doesn't hurt anybody to add it as a plugin.

Again, I already said the plugin is going to be incompatible with AutoOffhand which will cause more random complaints and troubles

**Comment by iBuyMountainDew** [comment]
They will never be enabled at the same time.

**Comment by l1ving** [comment]
> Plugin modules are able to annotate if they're incompatible with a module and then **not allow being enabled at the same time**.

**Comment by Xiaro** [comment]
> They will never be enabled at the same time.

:wonk: which will cause more random complaints and troubles

**Comment by l1ving** [comment]
If users can't read a popup that says "X is not compatible with Y" then that's their fault

**Comment by Xiaro** [comment]
> If users can't read a popup that says "X is not compatible with Y" then that's their fault

No, they will just ask you why is it incompatible and ask you to make them compatible instead

**Comment by l1ving** [comment]
:woeis:

**Comment by Doogie13** [comment]
> It is just gonna break compatibility and anticheat doesn't like switching items fast as well
ik its an old thread but this works on most servers ive used it on including 5b, cpvp.cc among others. a max switch thing (similar to gamesense) where you can only switch offhand item a few times until it wont switch it anymore (would ignore totems so you dont die easily) works well.
